### PR TITLE
Add chaos resource debugging to readme

### PR DESCRIPTION
**Pull request by benbp**

### CODE REVIEW

File: tools/stress-cluster/chaos/README.md [hunk 1]
```diff
@@ -523,16 +526,85 @@ a label like `job-name: <your job name>` in the drop down.
 
 See [Chaos Manifest](#chaos-manifest).
 
-## Running the example test with a network fault
+### Debugging chaos resources and events
+
+There are a few ways to check on the status of your chaos resources, after your stress test pod(s) reach a `Running` state.
+
+From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashbaord), select your stress test pods from the dropdown
```
Comment (yunhaoling) [inline tools/stress-cluster/chaos/README.md:533]:
```suggestion
From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashboard), select your stress test pods from the dropdown
```
typo:
```suggestion
From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashboard), select your stress test pods from the dropdown
```

Comment (yunhaoling) [inline tools/stress-cluster/chaos/README.md:533]: typo
```suggestion
From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashboard), select your stress test pods from the dropdown
```

**Review by yunhaoling — APPROVED** [review]

File: tools/stress-cluster/chaos/README.md [hunk 2]
```diff
@@ -507,6 +508,8 @@ Faults can be configured via kubernetes manifests or via the UI (which is a help
 
 ### Faults via Dashboard
 
+NOTE: The chaos mesh dashboard is just a helper for generating manifest under the hood. You can create and submit these directly as well. See the [docs](https://chaos-mesh.org/docs/chaos_experiments/networkchaos_experiment).
```
[the remark this answers is not in the document]
Comment (benbp) [reply tools/stress-cluster/chaos/README.md:511]:
```suggestion
NOTE: The chaos mesh dashboard is just a helper for generating manifest under the hood. You can create and submit these directly as well. See the [docs](https://chaos-mesh.org/docs/simulate-network-chaos-on-kubernetes/).
```

**Pull request merged**